x86_64-native-linuxapp-gcc-config

```diff
@@ -1,11 +1,19 @@
-# -*- cfg-sha: b5f82cd54947d45ceb377032450e026dc687f1cb43eed94c0aa2dd5893af261c
+# -*- cfg-sha: 4bfdb5adfc58968af6739d7098aa8a5ea16ae273a6a45571b926cf71826fd4c0
```

Comment:
I don't really know what all of these changes mean, but how did you arrive at them?

Reply:
I extracted the generated config from the SRPM artifact from the build